### ISSUE TITLE
2.2 POM is broken in Maven Central

### ISSUE THREAD

**Issue by ndrwdn** [paradoxical-io/dropwizard.swagger#12]
I'm attempting to use the latest version (2.2) and cannot. I believe this is because the POM in Maven Central has an incorrect version:
`<version>2.2${revision}</version>`

Thoughts on resolving this? Or perhaps a 2.3 release is imminent and this will be resolved then?

Thanks!

**Comment by devshorts** [comment]
Can you be more specific about what errors you are seeing? Is it that you can't pull it from maven or that there are other problems?  The maven version gets replaced at publish time, so the value in the Pom is actually a placeholder

**Comment by jakeswenson** [comment]
I experienced this problem the other day, i'll fix it 
Thanks for reporting @ndrwdn !

**Comment by ndrwdn** [comment]
@jakeswenson Thanks!

If it helps, with this in my build.gradle:
`compile 'io.paradoxical:dropwizard-swagger:2.2'`

I get this error:

```
* What went wrong:
Could not resolve all dependencies for configuration ':compileClasspath'.
> Could not resolve io.paradoxical:dropwizard-swagger:2.2.
  Required by:
      :dw-swagger-test:unspecified
   > Could not resolve io.paradoxical:dropwizard-swagger:2.2.
      > inconsistent module metadata found. Descriptor: io.paradoxical:dropwizard-swagger:2.2-SNAPSHOT Errors: bad version: expected='2.2' found='2.2-SNAPSHOT'
```

**Comment by jakeswenson** [comment]
That helped @ndrwdn. Sorry about all the problems. We just moved to a new build system. 
I spent a good chunk of time trying to resolve it, and all though i have fixed the issue with our pom for 2.3, 2.3 depends on a new module `dropwizard-bundles` and i need to resolve that as well first.
Expect a fix today, after i get our new maven plugin released. 

See details of what the plugin does here: https://github.com/paradoxical-io/resolved-pom-maven-plugin

Again thanks for the reporting with detailed errors, you rock!
